Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -520,7 +520,15 @@ public final class PackageBuilder {
                 if subpath == "" || subpath == "." {
                     return packagePath
                 }
-                let path = packagePath.appending(RelativePath(subpath))
+
+                let path: AbsolutePath
+
+                if subpath.hasPrefix("/") {
+                    path = AbsolutePath(subpath)
+                } else {
+                    path = AbsolutePath(subpath, relativeTo: packagePath)
+                }
```

Comment:
But we still need a method which can check if a path is valid or not. `AbsolutePath` or `RelativePath` will assert on invalid paths. For e.g. `.../test` is a completely invalid path.